### PR TITLE
support button now opens mail client with prefilled recipient and subject

### PR DESCRIPTION
**Pull request by dlhg**
- changed MenuItemButton component so that if incoming text prop is "Support", it renders an anchor tag leading to mailto link instead of a button
- added cypress test for this

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-8)

File: features/layout/sidebar-navigation/menu-item-button.tsx [hunk 1]
```diff
@@ -20,6 +20,25 @@ export function MenuItemButton({
   isCollapsed,
   rotateIcon = false,
 }: MenuItemProps) {
+  // Check if the button is the "support" button
+  if (text === "Support") {
```
Comment (pipergragg-siro) [inline features/layout/sidebar-navigation/menu-item-button.tsx:24]: 🔥

**Comment by dlhg** [comment]
Kind of a weird solution. I don't like adding the if check to the menu-item-buttons component, it feels clunky and counter to the spirit of React. Will check the solution vid after to see if there's a better way to do this and maybe rewrite this.

**Comment by dlhg** [comment]
Note to self - https://youtu.be/CZ9DyTdoDTI

**Pull request merged**